frontend/src/components/UserBookings.vue

```diff
@@ -1,31 +1,40 @@
 <template>
-  <div class="bg-gray-700 bg-opacity-50 rounded-3xl p-5 my-10" v-if="bookings.length">
+  <div class="bg-gray-700 bg-opacity-50 rounded-3xl py-10 px-20 my-10" v-if="bookings.length">
+    <h2 class="text-center mb-10">Bookings</h2>
+    <div
+        class="md:grid md:grid-cols-2 md:gap-4"
+        v-for="(booking, index) in bookings"
+        :key="booking.id"
+    >
+      <!-- Booking Details Column -->
+      <div>
+        <h3>{{ booking.schedule.movie.name }}</h3>
+        <p class="pt-3">
+          <strong>Cinema:</strong> {{ booking.schedule.cinema.name }} - {{ booking.schedule.cinemaHall.name }}
+        </p>
+        <p><strong>Duration:</strong> {{ booking.schedule.movie.duration }} minutes</p>
+        <p><strong>Average Rating:</strong> {{ averageRatings[booking.schedule.movie.id] || 'Loading...' }}%</p>
+        <p><strong>Showtime:</strong> {{ new Date(booking.schedule.localDateTime).toLocaleString().slice(0, 16) }}</p>
+        <p><strong>Booked Seats:</strong> {{ booking.bookedSeats.join(', ') }}</p>
+      </div>
 
-    <h2 class="text-center">Bookings</h2>
-    <div class="container mx-auto p-3 flex justify-evenly" v-for="(booking, index) in bookings" :key="booking.id">
-        <div>
-          <h4>{{ booking.schedule.movie.name }}</h4>
-          <h5 class="pt-3"><strong>Cinema:</strong> {{ booking.schedule.cinema.name }} -
-            {{ booking.schedule.cinemaHall.name }}</h5>
-          <p><strong>Duration:</strong> {{ booking.schedule.movie.duration }} minutes</p>
-          <p><strong>Average Rating:</strong> {{ averageRatings[booking.schedule.movie.id] || 'Loading...' }}%</p>
-          <p><strong>Showtime:</strong> {{ new Date(booking.schedule.localDateTime).toLocaleString() }}</p>
-          <p><strong>Booked Seats:</strong> {{ booking.bookedSeats.join(', ') }}</p>
-          <hr v-if="isLastBooking(booking)" class="mt-6">
-        </div>
+      <div class="flex justify-start md:justify-center mt-3 font-bold">
         <user-rating :movieId="booking.schedule.movie.id" @ratingSubmitted="updateAverageRating"></user-rating>
+      </div>
 
+      <!-- Divider Line After Each Booking Entry on Mobile -->
+      <div v-if="isNotLastBooking(booking)" class="col-span-1 border-t border-gray-500 my-6"></div>
     </div>
-
   </div>
 
   <p v-else>You have no bookings yet...</p>
 </template>
 
+
 <script setup>
 import {onMounted, ref} from 'vue';
 import UserRating from "@/components/UserRating.vue";
-const isLastBooking = (booking) => bookings.value.indexOf(booking) !== bookings.value.length - 1;
+const isNotLastBooking = (booking) => bookings.value.indexOf(booking) !== bookings.value.length - 1;
```

Comment:
Instead of looking for the index inside the method I suggest we could use the unused 'index' variable from the v-for at row 6 as a parameter and use that. 

```suggestion
const isNotLastBooking = (index) => index !== bookings.value.length - 1;
```